curiouslearning/FeedTheMonsterJS
# Fix Cr 175 - content changes for Somali

# Changes
- lang/somali/ftm_somali.json

# How to test
- When I play the development APK and open somali,
Then I should see the title as “Saaxiibkaa Quudii”.

Ref: [CR-175](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-175)


[CR-175]: https://curiouslearning.atlassian.net/browse/CR-175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
